contributors/devel/release/testing.md

```diff
@@ -0,0 +1,141 @@
+    - test a specific component - e.g. [federation, node]
+- *test infrastructure*
+  - not directly shown in the test grid
+  - libraries and infrastructure common across tests
+- *test failure*
+  - persistently failing test runs for a given test
+- *test flake*
+  - periodically failling test runs for a given test
+  
+## Ownership
+
+Each test must have an escalation point (email + slack).  The escalation point is responsible for
+keeping the test healthy.  Fixes for test failures caused by areas of ownership outside the
+responsibility of the escalation point should be coordinated with other teams by the
+test escalation point.
+
+Escalation points are expected to be responsive within 24 hours, and prioritize test failure
+issues over other issues.
+
+### test
+
+Each test must have an owning SIG or group that serves as the escalation point for flakes and failures.
+The name of the owner should be present in the test name so that it is displayed in the test grid.
+
+Owners are expected to maintain a dashboard of the tests that they own and
+maintain the test health.
+
+### test job
+
+Each test job must have an owning SIG or group that is responsible for the health of the test job.  The
+owner may also serve as an escalation point for issues impacting a test only in that specific test job
+(passing in other test jobs).  e.g. If a test only fails on aws or only on gke test jobs, the test job
+owner and test owner must identify the owner for resolving the failure.
+
+Owners of test jobs are expected to maintain a dashboard of the test jobs they own and
+maintain the test job health.
+
+### test infrastructure
+
+Issues with underlying test infrastructure (e.g. prow) should be escalated to XXXX.
+(Where are these escalated?)
+
+## Monitoring project wide test health
+
+Dashboards for Kubernetes release blocking test are present on the [test grid].
+
+The following dashboards are expected to remain healthy throughout the development cycle.
+
+- [release-master-blocking]
+  - Tests run against the master branch
+- [1.7-master-upgrade]
+  - TODO: Create these jobs and the dashboard
+  - Upgrade a cluster from 1.7 to the master branch and run tests
+- [1.6-master-upgrade]
+  - TODO: Create these jobs and the dashboard
+  - Upgrade a cluster from 1.6 to the master branch and run tests
+- [1.7-master-kubectl-skew]
+  - TODO: Create these jobs and the dashboard
+  - Run tests skewing the master and kubectl by +1/-1 version
+
+## Triaging ownership for test failures
+
+When a test is failing, it must be quickly escalated to the correct owner.  Tests that
+are left to fail for days or weeks become toxic and create noise in the system health
+metrics.
+
+The [build cop] is expected to ensure that the release blocking tests remain
+perpetually healthy by monitoring the test grid and escalating failures.
+
+*Tests without a responsive owner should be assigned a new owner or disabled.*
```

Comment:
Depends on who it is going to, and how often.  There are a lot of tools we can use - send to a slack channel with mentioning sig leads, create a GH issue, send an email, etc.  If the messages are ignored, we disable the test escalate to the kubernetes steering committee.